docs/modules/ROOT/pages/documentation/payara-server/server-configuration/certificate-management.adoc

```diff
@@ -0,0 +1,492 @@
+[[certificate-management]]
+= Integrated SSL/TLS Certificate Management
+
+_Since Payara Server 5.20.0_
+
+A set of commands to provide integrated management of SSL & TLS certificates in Payara Server.
+
+[[generate-self-signed-certificate]]
+== `generate-self-signed-certificate`
+
+*Usage*::
+`asadmin> generate-self-signed-certificate --target=instancename --listener=listenername --dn="DN1" --alternativenames="ALT1;ALT2;ALT3" alias`
+
+*Aim*::
+This command can generate a self-signed certificate for an instance, placing the resultant key pair in the target
+instance or listener's key and trust stores.
+
+If the instance or listener is configured to use the default key and trust store, the command will instead synchronise
+the instance with the DAS (under the assumption the certificate has been added to the default key and trust store of
+the DAS), since any certificates added to the instance stores would be lost upon next synchronisation.
+
+*Note*::
+While the command will not generate a certificate if one already exists in the target key store under the same alias,
+it will simply overwrite an entry using the same alias in the trust store.
```

Comment:
This sounds confusing to me. I'd read this as nothing changes but technically the entry gets rewritten as exactly the entry it as before. If so why does the docs have to state this detail?